### PR TITLE
Import DisplayablePorts from moby/moby source

### PR DESCRIPTION
**Pull request by vdemeester**
This has nothing to do in the `api` package for moby as it's just a
cli display function.

Related PR in moby/moby: https://github.com/moby/moby/issues/33798

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: cli/command/formatter/container.go [hunk 1]
```diff
@@ -6,12 +6,13 @@ import (
 	"strings"
 	"time"
 
+	"sort"
```
Comment (thaJeztah) [inline cli/command/formatter/container.go]: this should move up?

**Comment by thaJeztah** [comment]
Looks like CI (lint) isn't happy;

```
cli/command/formatter/container_test.go:649::warning: line is 216 characters (lll)
cli/command/formatter/container_test.go:649::warning: line is 216 characters (lll)
cli/command/formatter/container_test.go:649::warning: line is 216 characters (lll)
Exited with code 1

File: cli/command/formatter/container_test.go [hunk 2]
```diff
@@ -410,3 +410,250 @@ func TestContainerBackCompat(t *testing.T) {
...
+					Type:        "udp",
+				},
+			},
+			"80/tcp, 80/udp, 1024/tcp, 1024/udp, 1.1.1.1:1024->80/tcp, 1.1.1.1:1024->80/udp, 2.1.1.1:1024->80/tcp, 2.1.1.1:1024->80/udp, 1.1.1.1:80->1024/tcp, 1.1.1.1:80->1024/udp, 2.1.1.1:80->1024/tcp, 2.1.1.1:80->1024/udp",
+		},
+	}
+
+	for _, port := range cases {
+		actual := DisplayablePorts(port.ports)
+		if port.expected != actual {
```
Comment (dnephin) [inline cli/command/formatter/container_test.go]:
```go
assert.Equal()
```

File: cli/command/formatter/container_test.go [hunk 3]
```diff
@@ -410,3 +410,250 @@ func TestContainerBackCompat(t *testing.T) {
 		buf.Reset()
 	}
 }
+
+type ports struct {
+	ports    []types.Port
+	expected string
+}
+
+// DisplayablePorts
```
Comment (dnephin) [inline cli/command/formatter/container_test.go]: Comment can probably be removed :smile:

Comment (vdemeester) [inline cli/command/formatter/container_test.go]: oh lol, I just copied it, yeah it can :D

**Comment by vdemeester** [comment]
Updated, should be fixed 👼

File: cli/command/formatter/container_test.go [hunk 4]
```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// nolint: lllf
```
Comment (dnephin) [inline cli/command/formatter/container_test.go]: lllf?

Is this on the wrong function? It used to pass.

Comment (vdemeester) [inline cli/command/formatter/container_test.go]: ah dang it.. it's a typo on my part after running the lint 😱

Comment (vdemeester) [inline cli/command/formatter/container_test.go]: oh right 😛 Updated :)

**Comment by thaJeztah** [comment]
You forgot to update https://github.com/docker/cli/pull/236/files#r123762476 😇

**Comment by vdemeester** [comment]
@thaJeztah oups, done 🎐

**Pull request merged**